ci/workflows/main-cron.yml

```diff
@@ -550,6 +550,28 @@ steps:
     timeout_in_minutes: 25
     retry: *auto-retry
 
+  - label: "S3 sink batching test"
+    key: "s3-sink-batching-test"
+    command: "ci/scripts/s3-source-test.sh -p ci-release -s fs_sink_batch.py"
```

Comment:
We already have too many steps in the yaml for CI. Can we just use one step for all these tests?

Reply:
SGTM, let's reorganization and clean these CI test in separate pr.